src/components/ReactorEditor/useProcessJson.ts

```diff
@@ -46,7 +46,7 @@ export function useProessJson({
     })
 
     return () => jsWorker.terminate()
-  }, [editorName, jsWorker])
+  }, [model, jsWorker])
```

Comment:
Single `editorName` can have more than two remounts. Also there might be some time delay between `editorName` and `model` configured. Therefore every created model must be configured.